src/nvim/mapping.c

```diff
@@ -2396,6 +2408,7 @@ void modify_keymap(uint64_t channel_id, Buffer buffer, bool is_unmap, String mod
     KEY_TO_BOOL(script);
     KEY_TO_BOOL(expr);
     KEY_TO_BOOL(unique);
+    KEY_TO_BOOL(replace_keycodes);
```

Comment:
Should `replace_keycodes=true` with `expr=false` throw an error, or should it just be ignored?